CS2113 IP/src/main/java/Duke.java

```diff
@@ -0,0 +1,108 @@
+import java.util.Scanner;
+
+public class Duke {
+    public static void Greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        String horizontalLine = "________________________";
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(horizontalLine);
+    }
+
+    public static void listOperations() {
+        Scanner sc = new Scanner(System.in);
+        String horizontalLine = "________________________";
+        boolean isBye;
+        boolean isList;
+        boolean isDone;
+        boolean isTodo;
+        boolean isDeadline;
+        boolean isEvent;
+
+        Task[] taskList = new Task[100];
```

Comment:
Not sure if this name would be allowed but plural form should be used on names representing a collection of objects! So maybe it can be "tasks" instead of "taskList"?